3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -24,9 +24,9 @@ exercise file.
 const rollDice = require('../../helpers/pokerDiceRoller');
 
 function rollTheDices() {
-  // TODO Refactor this function
   const dices = [1, 2, 3, 4, 5];
-  return rollDice(1);
+  const dice = dices.map((dice) => rollDice(dice));
+  return Promise.all(dice);
```

Comment:
woo nice use of `map` 👏 but you can be even ~lazier~ more efficient
```suggestion
  return Promise.all(dices.map(rollDice));
```